### PR TITLE
fixes rust builder and simplifies process

### PR DESCRIPTION
**Pull request by DylanVerstraete**
Resolves: https://github.com/threefoldtech/jumpscaleX/issues/332

Installs rust with dynamic platform installer.

### CODE REVIEW

File: Jumpscale/builder/runtimes/BuilderRust.py [hunk 1]
```diff
@@ -8,16 +8,48 @@ class BuilderRust(j.builder.system._BaseClass):
...
     @builder_method()
-    def install(self):
+    def build(self):
         self.profile_sandbox_select()
 
         # Will download and run the correct version of rustup-init executable for your platform
         self._execute(
             "curl {} -sSf | sh -s -- -y".format(self.DOWNLOAD_URL)
         )
+        self._copy(self.DIR_BUILD, "{DIR_BIN}")
```
Comment (BolaNasr) [inline Jumpscale/builder/runtimes/BuilderRust.py]: should be in install not in build

File: Jumpscale/builder/runtimes/BuilderRust.py [hunk 2]
```diff
@@ -8,16 +8,48 @@ class BuilderRust(j.builder.system._BaseClass):
...
+
         self._execute("source $HOME/.cargo/env")
```
Comment (BolaNasr) [inline Jumpscale/builder/runtimes/BuilderRust.py:22]: why you execute this here ?

File: Jumpscale/builder/runtimes/BuilderRust.py [hunk 3]
```diff
@@ -8,16 +8,48 @@ class BuilderRust(j.builder.system._BaseClass):
...
-        self._execute("echo  'export PATH='$HOME/.cargo/bin:$PATH'' >> ~/.bashrc ")
+        self._execute("echo  'export PATH='/root/.cargo/bin:$PATH'' >> ~/.bashrc ")
+
+    @builder_method()
+    def install(self):
```
Comment (BolaNasr) [inline Jumpscale/builder/runtimes/BuilderRust.py:26]: should here copy binary from builder path to ```/sandbox/bin```

Comment (DylanVerstraete) [reply Jumpscale/builder/runtimes/BuilderRust.py:26]: `self._copy(self.DIR_BUILD, "{DIR_BIN}")` ?

Comment (BolaNasr) [reply Jumpscale/builder/runtimes/BuilderRust.py:26]: yes

**Comment by BolaNasr** [comment]
LGTM

**Review by BolaNasr — APPROVED** [review]

**Pull request merged**